aip/general/0145.md

```diff
@@ -48,6 +48,25 @@ Exclusive end values are preferable for the following reasons:
 - It is easier to reason about abutting ranges: `[0, x), [x, y), [y, z)`, where
   values are chainable from one range to the next.
 
+### Use of `google.type.Interval`
+
+The [`google.type.Interval`][interval] type (in the common protos described in
+[AIP-213][]) represents a range between two timestamps, with an inclusive start
+value and exclusive end value as described above.
+
+Ranges between two timestamps which conform to the expectations of the
+`Interval` message **should** usually use this rather than having separate
+start and end fields. This allows client code to be written against the
+`Interval` message (such as checking whether a given timestamp occurs within
+the interval) and reused across multiple intervals in the same API, or even
+across multiple APIs.
+
+This is only a weak recommendation, and APIs **may** use start and end
+timestamp fields instead. In particular, if a message within an API is inherently
```

Comment:
I'm afraid I don't understand the sentence starting, "In particular, ". Could you take a stab at reworking, clarifying, or explain the meaning and I would do the same?

Reply:
It's trying to describe the rule of thumb suggested in the YAQS question this PR came from. I can ping you the link internally if you want.